### PR TITLE
Ivm top10 vuln assets to slack

### PR DESCRIPTION
**Pull request by emartin-merrill-r7**
## Requirements

Developers, verify you have completed the following items by checking them off:

### Checklist
- [x] Workflow/Snippet contains title and detailed description
- [x] Workflow/Snippet title should follow general [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [x] User friendly step names in [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [x] User friendly step descriptions where needed
- [x] Workflow contains at least 1 artifact with well formatted information
- [x] Directory created and named after the title the snippet or workflow e.g. `workflows/Basic_Domain_Enrichment_Report/`, `snippets/Domain_Enrichment`
- [x] Workflow `.icon` file is in the workflow's directory e.g. `workflows/Basic_Domain_Enrichment_Report/Basic_Domain_Enrichment_Report.icon`
- [x] Take PNG screenshots of the workflow or snippet e.g.  https://market.komand.com/workflows/ross/chatops-github-commands/1.0.0, https://market.komand.com/snippets/jschipp/basic-domain-enrichment-report/0.1.0
- [x] Create a directory called `screenshots` in your workflow's/snippet's directory
- [x] Includes PNG screenshots of the workflow in canvas, job, artifact in the `screenshots` directory e.g.  `workflows/Basic_Domain_Enrichment_Report/screenshots/`
- [x] PNG screenshot file names should conform to style e.g. `workflow1.png`, `artifact1.png`, `job.png`
- [x] Workflow verified as working through WUI
- [x] Credentials are not exported and stored in git
- [x] `workflow.spec.yaml` file with metadata accompanies workflow
- [x] Workflow documentation is provided in `help.md`
- [x] Directory structure is followed, see https://github.com/rapid7/insightconnect-workflows/blob/master/README.md

### Assessment

Post your results with screenshots in this PR using the following style:

### UI

chatops_example_1

#### Workflow Builder

workflow_example_1
workflow_example_2

### CODE REVIEW

File: workflows/InsightVM_Top_10_Vuln_Assets_to_Slack/help.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Description
+
+This workflow returns the top 10 most vulnerable assets in each site in InsightVM
```
Comment (jschipp-r7) [inline workflows/InsightVM_Top_10_Vuln_Assets_to_Slack/help.md:3]:
```suggestion
This workflow returns the top 10 most vulnerable assets in each site in InsightVM.
```

File: workflows/InsightVM_Top_10_Vuln_Assets_to_Slack/help.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Description
+
+This workflow returns the top 10 most vulnerable assets in each site in InsightVM.
+
```
Comment (jschipp-r7) [inline workflows/InsightVM_Top_10_Vuln_Assets_to_Slack/help.md:4]: Add example, see: https://github.com/rapid7/insightconnect-workflows/blob/master/workflows/Lookup_Vuln_From_Slack/help.md

Sample Slack Trigger Command:

```
@Slack bot list most vulnerable assets
```

File: workflows/InsightVM_Top_10_Vuln_Assets_to_Slack/workflow.spec.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+extension: workflow
+products: ["insightconnect"]
+name: InsightVM_Top_10_Vuln_Assets_to_Slack
+title: "Most Vulnerable Assets in Sites to Slack"
+description: "This workflow returns the top 10 most vulnerable assets in each site in InsightVM"
```
Comment (jschipp-r7) [inline workflows/InsightVM_Top_10_Vuln_Assets_to_Slack/workflow.spec.yaml:5]: Period at end of sentence

File: workflows/InsightVM_Top_10_Vuln_Assets_to_Slack/help.md (hunk 2 above)
Comment (emartin-merrill-r7) [reply workflows/InsightVM_Top_10_Vuln_Assets_to_Slack/help.md:4]: changed

**Comment by jschipp-r7** [comment]
Title validator is a FP.  LGTM 👍 

```
$ icon-validate .
[*] Validating workflow at .

[*] Running Integration Validators...
[*] Executing validator WorkflowFilesValidator
[*] Executing validator WorkflowHelpValidator
[*] Executing validator WorkflowChangelogValidator
[*] Executing validator WorkflowVendorValidator
[*] Executing validator WorkflowVersionValidator
[*] Executing validator WorkflowExtensionValidator
[*] Executing validator WorkflowSupportValidator
[*] Executing validator WorkflowPNGHashValidator
[*] Executing validator WorkflowICONFileNameValidator
[*] Executing validator WorkflowScreenshotValidator
[*] Executing validator WorkflowTitleValidator
Validator WorkflowTitleValidator failed!
Title contains a lowercase 'in' when it should not.
[*] Executing validator WorkflowDescriptionValidator
[*] Executing validator WorkflowNameValidator
[*] Executing validator WorkflowProfanityValidator
[*] Executing validator WorkflowHelpPluginUtilizationValidator
[*] Executing validator WorkflowICONFileValidator
[*]Workflow failed validation!

----
[*] Total time elapsed: 19.220000000000002ms
```

File: workflows/InsightVM_Top_10_Vuln_Assets_to_Slack/help.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
+# Description
+
+This workflow returns the top 10 most vulnerable assets in each site in InsightVM.
```
Comment (zyoutz-r7) [inline workflows/InsightVM_Top_10_Vuln_Assets_to_Slack/help.md:3]: Could we add a bit more details as to why someone would want to use this?  eg. `This allows teams to quickly identify and communicate assets that need immediate attention to teams within their environment with a tool that is easily accessible - Slack!`

File: workflows/InsightVM_Top_10_Vuln_Assets_to_Slack/InsightVM_Top_10_Vuln_Assets_to_Slack.icon [hunk 5]
```diff
@@ -0,0 +1,1279 @@
...
+  "kom": {
+    "komandVersion": "1.48.1-0-g787c7",
+    "komFileVersion": "2.0.0",
+    "exportedAt": "2020-04-06T04:13:06.958598442Z",
+    "workflowVersions": [
+      {
+        "name": "Most Vulnerable Assets in Sites to Slack",
+        "type": "runnable",
+        "version": "",
+        "description": "",
```
Comment (jschipp-r7) [inline workflows/InsightVM_Top_10_Vuln_Assets_to_Slack/InsightVM_Top_10_Vuln_Assets_to_Slack.icon:11]: Missing workflow description

File: workflows/InsightVM_Top_10_Vuln_Assets_to_Slack/help.md (hunk 4 above)
Comment (emartin-merrill-r7) [reply workflows/InsightVM_Top_10_Vuln_Assets_to_Slack/help.md:3]: I think this is a great suggestion for Version 2 - This version is quite restrictive.

**Pull request merged**